src/styles/_catalog.scss

```diff
@@ -1,22 +1,28 @@
 .catalog {
-  display: flex;
-  flex-wrap: wrap;
+  display: grid;
+  grid-template-columns: repeat(1, 200px);
+  gap: 48px;
+  justify-content: center;
+  padding-bottom: 48px;
 
-  max-width: 944px;
-  margin: 0 auto;
+  @media (min-width: (488px)) {
+    grid-template-columns: repeat(2,200px);
+  }
+
+  @media (min-width: (768px)) {
+    grid-template-columns: repeat(3,200px);
+  }
+
+  @media (min-width: (1024px)) {
+    grid-template-columns: repeat(4,200px);
+  }
```

Comment:
You can make a loop to create all media queries. Just create a correctly mapped list to iterate.